code/game/machinery/syndicatebomb.dm

```diff
@@ -77,7 +77,8 @@
 		update_icon()
 		try_detonate(TRUE)
 	//Counter terrorists win
-	else if(!active || defused)
+	else if(defused)
+		active = FALSE
 		if(defused && (payload in src))
 			payload.defuse()
 			countdown.stop()
```

Comment:
This one and the line below should be in the scope of the defused if.
It might be nice to make a proc to deactivate the bomb since it's done twice now. Once here and once on line 49-53

Reply:
When discussing in coding chat, some us wondered that if we did merge those two if's, what state would we leave the bomb in, if it did not have a payload? As for making that proc, i might need a little help with that. Alternatively, only keep payload.defuse(), and leave the rest to be done on next tick of process() ?

Reply:
Also a possibility to do that yeah. Feel free to ping me if you choose to do it the proc way. The way you suggested also works in this case

Reply:
Chose to cleanup, and let payload and countdown be run on next tick of process instead.